externs/browser/html5.js

```diff
@@ -3570,9 +3570,64 @@ HTMLDataListElement.prototype.options;
  */
 function HTMLOutputElement() {}
 
-// TODO(jakubvrana): Add HTMLOutputElement properties.
+/**
+ * @const
+ * @type {!DOMTokenList}
+ */
+HTMLOutputElement.prototype.htmlFor;
+
+/**
+ * @type {HTMLFormElement}
+ */
+HTMLOutputElement.prototype.form;
+
+/**
+ * @type {string}
+ */
+HTMLOutputElement.prototype.name;
+
+/**
+ * @const
+ * @type {!string}
+ */
+HTMLOutputElement.prototype.type = "output";
```

Comment:
In general, externs should not have implementations. If they do - it's a mistake. It leads people to believe that the code actually executes.